Bottom: 4bac2b3484c623ea72a68bb79d92359459164824
Top:    6af375d37de5bc18f66c1a29049ac4b0a57ae43e
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-04-27 21:15:21 +0100

Refresh of temporary-comment-out-some-cal

---

diff --git a/src/hid/gtk/gtkhid-main.c b/src/hid/gtk/gtkhid-main.c
index 0df478d..919cec7 100644
--- a/src/hid/gtk/gtkhid-main.c
+++ b/src/hid/gtk/gtkhid-main.c
@@ -1893,9 +1893,9 @@ PanAction (int argc, char **argv, int x, int y)
     }
   else if (x == on_x && y == on_y)
     {
-      ghid_show_crosshair (FALSE);
+      //ghid_show_crosshair (FALSE);
       ghidgui->auto_pan_on = !ghidgui->auto_pan_on;
-      ghid_show_crosshair (TRUE);
+      //ghid_show_crosshair (TRUE);
     }
 
   return 0;
